### DISCUSSION TITLE
Default Tauri App

### DISCUSSION THREAD

**Discussion by Brandon-Chan03** [tauri-apps/tauri#6648] [Q&A] [answered]
when you use "npm create tauri-app@latest" you get the default Tauri app. It has a small text field with a submit button which then invokes a rust function and sends back a greeting with what was inputted in the text field. I wanted to know where the code was that persists the previously submitted inputs, as when you close the application and re-open it; it shows a drop-down of previous submissions.

**Comment by FabianLars** [comment, marked as answer]
That's handled by the webviews themselves, similar to how it works in the browser. If you want to disable it you can add `autocomplete="off"` to the input fields, but there's no way to manually control it's contents (or save them).

**Reply by Brandon-Chan03** [reply]
Thank you so much for the reply! This makes so much more sense. I’ve been using Tauri for about a week but never honestly noticed because I started coding out the front end before deploying the dev environment and never saw it.
